app/controllers/form_elements_controller.rb

```diff
@@ -6,8 +6,7 @@ def create
 
     respond_to do |format|
       if @form_element.valid?
-        format.js
```

Comment:
I took this out yesterday when merging because this line broke all the event bindings on the form editor when I added a form element. It all works now, but if there's another way to make it work, I'm in.